zk-token-sdk/src/zk_token_elgamal/pod/mod.rs

```diff
@@ -0,0 +1,76 @@
+mod auth_encryption;
+mod elgamal;
+mod instruction;
+mod pedersen;
+mod range_proof;
+mod sigma_proofs;
+
+use {
+    crate::zk_token_proof_instruction::ProofType,
+    num_traits::{FromPrimitive, ToPrimitive},
+    solana_program::instruction::InstructionError,
+};
+pub use {
+    auth_encryption::AeCiphertext,
+    bytemuck::{Pod, Zeroable},
```

Comment:
Incidentally, why do we re-export these traits?

Reply:
The bytemuck version on the monorepo and spl side do not always match, so I had some issues before without re-exporting these traits. I can't seem to reproduce it or recall what the precise issue was right now though ... :confused: